### ISSUE TITLE
AppVeyor failure being misreported as pass

### ISSUE THREAD

**Issue by rtfeldman** [rtfeldman/node-test-runner#333]
Some sort of tmpfile issue - scroll all the way to the end, when it's testing flags.js, to see it: https://ci.appveyor.com/project/rtfeldman/node-test-runner/builds/21730936/job/a0y9k04p7rtvw37x

**Comment by harrysarson** [comment]
What is happening is that in `tests/flags.js` there is a test `should not allow command injection` added by me in #330.

When implementing this test I added a `console.log` for debugging purposes which I forgot to remove when submitting the pull request.
The command is _meant_ to fail (as the command includes command injection) so AppVeyor is correct in reporting it as a pass.

However, we should not be logging the output of the run. My pull request #335 addresses this.

### TLDR;

This is not a bug and this issue can be closed.
